src/main/java/io/debezium/connector/informix/InformixConnection.java

```diff
@@ -42,6 +45,7 @@ public class InformixConnection extends JdbcConnection {
     private static final String GET_DATABASE_NAME = "select dbinfo('dbname') as dbname from systables where tabid = 1";
 
     private static final String GET_MAX_LSN = "select uniqid, used as logpage from sysmaster:syslogs where is_current = 1";
+    private static final String GET_MIN_LSN = "select uniqid, 0 as logpage from sysmaster:syslogs where is_current = 1";
```

Comment:
This will only select the start page of the current log file. To select the oldest avaliable LSN `select min(uniqid) as uniqid , 0 as logpage from sysmaster:syslogs` should do the trick...

Reply:
Thank you @nrkljo for the tip